docs/src/main/paradox/scala/http/migration-guide/index.md

```diff
@@ -7,6 +7,6 @@
 
 * [migration-from-spray](migration-from-spray.md)
 * [migration-guide-2.4.x-10.0.x](migration-guide-2.4.x-10.0.x.md)
-* [migration-within-10.0.x-Api-May-Change](migration-within-10.0.x-Api-May-Change.md)
+* [migration-guide-10.0.x](migration-guide-10.0.x.md)
```

Comment:
No strong feelings, now it matches the pattern. I used this name to show clearly that it's only about changes on `ApiMayChange`.